lib/proxy/http_download.rb

```diff
@@ -3,35 +3,56 @@
 module Proxy
   class HttpDownload < Proxy::Util::CommandTask
     include Util
-    DEFAULT_READ_TIMEOUT = 60
     DEFAULT_CONNECT_TIMEOUT = 10
-    DEFAULT_DNS_TIMEOUT = 10
 
     def initialize(src, dst, read_timeout = nil, connect_timeout = nil, dns_timeout = nil, verify_server_cert = false)
```

Comment:
It looks like read_timeout and dns_timeout are no longer used. Shouldn't we drop those instead? Or at least issue a deprecation warning?

Reply:
Hmm I just realized these are exposed as settings, this is from 2018 so it is likely in active use. Deprecation is then the best way to do this.